### PR TITLE
Test using a smaller dt to fix linear_nonlinear_krylov_tests.jl

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

File: test/algconvergence/linear_nonlinear_krylov_tests.jl [hunk 1]
```diff
@@ -57,13 +57,13 @@ let N = 20
     end
 
     @testset "EPIRK" begin
-        dt = 0.05
+        dt = 0.005
```
Comment (ChrisRackauckas) [inline test/algconvergence/linear_nonlinear_krylov_tests.jl:60]:
```suggestion
        dt = 0.001
```

**Comment by vpuri3** [comment]
The skipped test below also passes when you bring down dt from `1 / 16` to `1 / 128`.  Don't forget to change it on both lines 86, and 91. 

https://github.com/SciML/OrdinaryDiffEq.jl/blob/dade7ab3d03d721de620d0d6aa225c36958edf01/test/algconvergence/linear_nonlinear_krylov_tests.jl#L84-L96

**Comment by vpuri3** [comment]
Based on the description in https://github.com/SciML/OrdinaryDiffEq.jl/issues/1956 I thought we only looking at the `@test_skip`s. I went back to `v6.51.0` (before the scimlops change) and these tests were already marked `@test_broken`. 

https://github.com/SciML/OrdinaryDiffEq.jl/blob/v6.51.0/test/algconvergence/linear_nonlinear_krylov_tests.jl#L59-L82

Nevertheless, I brought down `dt` to `1e-4` and almost all of them passed. Can you do the same here?

File: test/algconvergence/linear_nonlinear_krylov_tests.jl [hunk 2]
```diff
@@ -73,11 +73,11 @@ let N = 20
 
         sol = solve(prob, EPIRK5s3(); dt = dt, reltol = tol)
         sol_ref = solve(prob, Tsit5(); reltol = tol)
-        @test_broken isapprox(sol(1.0), sol_ref(1.0); rtol = tol)
+        @test isapprox(sol(1.0), sol_ref(1.0); rtol = tol)
```
Comment (ChrisRackauckas) [inline test/algconvergence/linear_nonlinear_krylov_tests.jl:76]:
```suggestion
        @test_broken isapprox(sol(1.0), sol_ref(1.0); rtol = tol)
```

File: test/algconvergence/linear_nonlinear_krylov_tests.jl [hunk 3]
```diff
@@ -73,11 +73,11 @@ let N = 20
...
 
         sol = solve(prob_ip, EPIRK5s3(); dt = dt, reltol = tol)
         sol_ref = solve(prob_ip, Tsit5(); reltol = tol)
-        @test_broken isapprox(sol(1.0), sol_ref(1.0); rtol = tol)
+        @test isapprox(sol(1.0), sol_ref(1.0); rtol = tol)
```
Comment (ChrisRackauckas) [inline test/algconvergence/linear_nonlinear_krylov_tests.jl:80]:
```suggestion
        @test_broken isapprox(sol(1.0), sol_ref(1.0); rtol = tol)
```

**Comment by vpuri3** [comment]
```julia
[ Info: Exp4, dt = 0.0001
isapprox(sol(1.0), sol_ref(1.0); rtol = tol) = true
Exp4
[ Info: EPIRK4s3A, dt = 0.0001
isapprox(sol(1.0), sol_ref(1.0); rtol = tol) = true
EPIRK4s3A
[ Info: EPIRK4s3B, dt = 0.0001
isapprox(sol(1.0), sol_ref(1.0); rtol = tol) = false 
EPIRK4s3B
[ Info: EXPRB53s3, dt = 0.0001
isapprox(sol(1.0), sol_ref(1.0); rtol = tol) = true
EXPRB53s3
[ Info: EPIRK5P1, dt = 0.0001
isapprox(sol(1.0), sol_ref(1.0); rtol = tol) = true
EPIRK5P1
[ Info: EPIRK5P2, dt = 0.0001
isapprox(sol(1.0), sol_ref(1.0); rtol = tol) = true
EPIRK5P2
[ Info: EPIRK5s3 oop, dt = 0.0001
isapprox(sol(1.0), sol_ref(1.0); rtol = tol) = false 
[ Info: EPIRK5s3 iip, dt = 0.0001
isapprox(sol(1.0), sol_ref(1.0); rtol = tol) = true
```

**Comment by vpuri3** [comment]
Only `EPIRK4s3B`, `EPIRK5s3` (only OOP) are failing with `dt = 1e-4`.

**Comment by vpuri3** [comment]
And both pass with `dt = 1e-5`.

File: test/algconvergence/linear_nonlinear_krylov_tests.jl [hunk 4]
```diff
@@ -57,13 +57,13 @@ let N = 20
     end
 
     @testset "EPIRK" begin
-        dt = 0.05
+        dt = 0.001
```
Comment (ChrisRackauckas) [inline test/algconvergence/linear_nonlinear_krylov_tests.jl:60]:
```suggestion
        dt = 1e-5
```

**Comment by ChrisRackauckas** [comment]
The ones that were marked broken before were marked again.

but  EPIRK worked before with `dt = 0.05; tol=1e-5`, and IIP works with it too. That's a pretty major regression.

**Comment by vpuri3** [comment]
yeah, this is a weird error. I added a print statement to OOP update_func, and it is being called. Maybe it's not being assigned correctly.

Lemme look again over the weekend.

**Pull request closed without merging**